bincrafters_conventions/actions/update_c_recipe_references.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+
+def update_c_recipe_references(main, file):
+    """ This update script updates Conan references, mostly
+
+    :param file: Conan file path
+    """
+
+    if not os.path.isfile(file):
+        return False
+
+    references_updated = False
+
+    references = {
+        "zlib/1.2.8@conan/stable": "zlib/1.2.11",
```

Comment:
maybe if we use wildcars/regex here and below it will be less? like `OpenSSL/1.0.2*@conan/stable` -> `openssl/1.0.2t`